ambari-common/src/main/python/resource_management/libraries/functions/tar_archive.py

```diff
@@ -78,4 +91,4 @@ def mode(archive):
   elif archive.endswith('.tar.bz2') or archive.endswith('.tbz'):
     return 'r:bz2'
   else:
-    raise ValueError("Could not extract `%s` as no appropriate extractor is found" % archive)
\ No newline at end of file
+    raise ValueError("Could not extract `%s` as no appropriate extractor is found" % archive)
```

Comment:
since we moved to python 2.7, lets use `string.format` instead  of `%`

Reply:
The only change in this line is newline at the end.  Plus this will need to be backported to branch-2.6, so I'd prefer to keep differences minimal.